source/assets/scripts/main.js

```diff
@@ -95,6 +95,7 @@ function taskCount() {
 
 /**
  * load db from local storage
```

Comment:
Be more specific on db we found: may be just add "load to do list DB (users & their tasks)" sounds better